css/main.css

```diff
@@ -1,3 +1,345 @@
 /* Add your CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+:root {
+  --black: #000;
+  --white: #fff;
+  --red-brown: #a05941;
+  --light-grey: #ededee;
+  --green-black: #133032;
+}
+
+body {
+  background-color: var(--light-grey);
+  color: var(--red-brown);
+  margin: 0;
+  font-size: 20px;
+  font-family: "Fjalla One", sans-serif;
```

Comment:
There is a bit of a mixup with the font families - compare you headings with the design, and then compare the body copy with the design. Do you see the font families have been inverted? Swap them around and you'll be much closer to the design 😉